packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -679,7 +679,10 @@ export const useArbTokenBridge = (
       for (const address in bridgeTokens) {
         const token = bridgeTokens[address]
         if (!token) continue
-        if (token.listID === listID) {
+
+        token.listIds.delete(listID)
+
+        if (!token.listIds.size) {
```

Comment:
is this checking for size === 0?
if it is, i'd prefer it saying === 0 than !size for readability

Reply:
yes, `.size` is the set equivalent of array's `.length`,

Reply:
yea, i think ! works but === 0 is clearer